chase/frontend/app/page.tsx

```diff
@@ -31,6 +55,28 @@ export default function Home() {
               onClick={() => router.push("/login/chair")}
             />
           </div>
+          <span>
+            backendAcceptsOurAuth:
+            {backendAcceptsOurAuth && "yes"}
+            {!backendAcceptsOurAuth && "nonono"}
+          </span>
+          <span>
+            authenticated:
+            {auth.authenticated && "yes"}
+            {!auth.authenticated && "nonono"}
```

Comment:
"nonono"? xD

Reply:
This was my demo code :D We should probably not ship this to prod, although it might amuse me